settings.gradle

```diff
@@ -99,16 +109,16 @@ setupProject(':lib-jexl', 'components/lib/jexl', 'Javascript Expression Language
 // Tooling
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-setupProject(':tooling-lint', 'components/tooling/lint', 'Custom Lint checks for using and writing components.')
+setupProject(':tooling-lint', 'components/tooling/lint', 'Custom Lint checks for using and writing components.', false)
```

Comment:
Does `Groovy` allow to use named parameters to make it more obvious what `false` means?

Reply:
Unfortunately not, no.
It _does_ allow [named parameters](http://docs.groovy-lang.org/latest/html/documentation/#_named_arguments), but that'd require `setupProject` to take a single parameter (a map), but I believe that that will reduce usability

Reply:
Okay, no problem. With Gradle 5 I'll try to migrate all that nasty Groovy to Kotlin :)